nbautoexport/utils.py

```diff
@@ -6,15 +6,18 @@
 from typing import List
 from warnings import warn
 
+if sys.version_info[:2] >= (3, 8):
+    import importlib.metadata as importlib_metadata
+else:
+    import importlib_metadata
+
 from pydantic import BaseModel
 from nbconvert.exporters import get_export_names, get_exporter
 import nbformat
 from jupyter_core.application import JupyterApp
 
-from nbautoexport._version import get_versions
-
 
-__version__ = get_versions()["version"]
+__version__ = importlib_metadata.version(__name__.split(".", 1)[0])
```

Comment:
can you help me understand what `.split(".", 1)[0]` is doing? trying to understand if there are edge cases here with minor versions or something

Reply:
This is all inside the call to `importlib_metadata.version(...)`.

In this case, `__name__` is `nbautoexport.utils` and this is just to grab `nbautoexport` from that. We could also use `__name__.partition(".")[0]` (which is maybe only slightly more clear), or just hardcode this to `"nbautoexport"`.

Reply:
i have a slight preference for hardcoding to "nbautoexport"